config/config.php

```diff
@@ -326,3 +326,10 @@ functionality to PhpReport
  * To be used in combination with ISSUE_TRACKER_LINKS_URL, see its entry above.
  */
 define('ISSUE_TRACKER_LINKS_TEXT', serialize(array('Report an issue')));
+
+/* CalDav integration */
+define('CALENDAR_URL', '');
+define('CALENDAR_ID', '');
```

Comment:
I don't know what this parameter means or how to get it.

Reply:
This is an odd flow indeed. So even though we pass the calendar url to a specific calendar, when connecting to the calendar the server, it returns **all** the calendars for that user, that's why we need the url and the ID.

I don't know if that's the standard for every caldav server, but in our case (Sogo), the calendar ID is the last parameter of the calendar URL, for example:

From `https://calendar-beta.igalia.com/SOGo/dav/igalia-holidays/Calendar/50E03-619E9800-B9-7E0D1A80/` the ID is `50E03-619E9800-B9-7E0D1A80`.